tests/Uri/BuildTest.php

```diff
@@ -8,15 +8,23 @@ class BuildTest extends \PHPUnit\Framework\TestCase
 {
     /**
      * @dataProvider buildUriData
+     *
+     * @param string $value
      */
-    public function testBuild($value)
+    public function testBuild($value): void
     {
+        /** @var array<string, int|string> $parsedUrl */
+        $parsedUrl = parse_url($value);
+        $this->assertIsArray($parsedUrl);
         $this->assertEquals(
             $value,
-            build(parse_url($value))
+            build($parsedUrl)
         );
     }
 
+    /**
+     * @return array<int, array<int, string>>
```

Comment:
I guess this should/could be

```suggestion
     * @return list<list<string>>
```

Reply:
That will work for `phpstan` 
My PHPstorm IDE does not like it - I guess I need to find the extension that teaches it abut that sort of "extended PHPdoc"